pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -514,7 +516,7 @@ func TestDesiredRouterDeploymentSpecAndNetwork(t *testing.T) {
 		{"ROUTER_H1_CASE_ADJUST", true, "Host,Cache-Control"},
 		{"ROUTER_CANONICAL_HOSTNAME", true, "router-" + ic.Name + "." + ic.Status.Domain},
 		{"ROUTER_LOG_FACILITY", false, ""},
-		{"ROUTER_LOG_MAX_LENGTH", false, ""},
+		{"ROUTER_LOG_MAX_LENGTH", true, "8192"},
```

Comment:
See https://github.com/openshift/cluster-ingress-operator/pull/900/files/e965950e1d1bb700c455d019389a28104a75c964#r1175768966

Reply:
E.g. don't do this here.  Add a new test elsewhere, which tests the change from default to a new value.

Reply:
See `TestDeploymentConfigChanged` for examples.